src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/_params.py

```diff
@@ -5,81 +5,97 @@
 from azure.cli.core.sdk.util import ParametersContext
 from azure.cli.core.util import get_json_object
 
-# For some commands we take JSON strings as possible
-with ParametersContext(command="sf application create") as c:
-    c.register("parameters", ("--parameters",), type=get_json_object,
-               help="JSON encoded list of application parameters.")
 
-with ParametersContext(command="sf application create") as c:
-    c.register("metrics", ("--metrics",), type=get_json_object,
-               help="JSON encoded list of application metrics and their \
-               descriptions.")
+# Global parameters
+# Documentation here can be omitted from command specification
+with ParametersContext(command="sf") as c:
+    c.argument("timeout", options_list=("--timeout", "-t"),
+               type=int,
+               help=("The server timeout for performing the operation, "
```

Comment:
`_help.py` is not for command groups only. It can be used for commands and it's an actually recommended approach. Regarding documenting parameter, you can refer to the sample here: https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py#L25.